lib/pghero/methods/citus.rb

```diff
@@ -0,0 +1,29 @@
+module PgHero
+  module Methods
+    module Citus
+      def citus_version
+        @citus_version ||= select_one("SHOW citus.version")
+      end
+
+      def citus_available?
```

Comment:
Who uses this function?

Reply:
It is not used for now so I will remove it.